package.json

```diff
@@ -4,13 +4,14 @@
   "version": "0.0.0",
   "main": "dist/index.html",
   "scripts": {
-    "start": "./node_modules/.bin/gulp",
-    "build": "./node_modules/.bin/gulp build",
-    "test": "./node_modules/.bin/gulp test",
-    "serve": "./node_modules/.bin/gulp serve",
-    "deploy": "./node_modules/.bin/gulp deploy"
```

Comment:
In that case you'd need to remove the global one because it would trump this one. I'm in favor of never using global node modules.